monkestation/code/modules/cassettes/cassette_approval.dm

```diff
@@ -73,26 +73,22 @@ GLOBAL_LIST_INIT(cassette_reviews, list())
 	var/obj/item/device/cassette_tape/submitted_tape
 
 	var/action_taken = FALSE
+	var/verdict = "NONE"
 
 /datum/cassette_review/Destroy(force)
 	. = ..()
-	QDEL_LIST(cassette_data)
+	if(cassette_data)
+		QDEL_LIST(cassette_data)
 	submitter = null
-
-	GLOB.cassette_reviews["[id]"] -= src
-	GLOB.cassette_reviews -= id
+	if(id && (id in GLOB.cassette_reviews))
+		GLOB.cassette_reviews[id] = null // Nullify the entry
+		GLOB.cassette_reviews -= id // Remove the key
```

Comment:
```suggestion
		GLOB.cassette_reviews -= id // Remove the key
```
don't need to set it to null if you're removing it anyways

Reply:
Fixed though wanted to ask about the lines above. QDEL_LIST runtimes when I call to destroy the entries at the end of the shift.